specification/communication/data-plane/CallingServer/preview/2021-08-30-preview/communicationservicescallingserver.json

```
@@ -1138,13 +1138,16 @@
           "type": "string"
         },
         "recordingContentType": {
-          "$ref": "#/definitions/RecordingContentType"
+          "$ref": "#/definitions/RecordingContentType",
+          "description": "Optional, AudioVideo by Default"
```

grammar nit: (incl. casing of the value)
```suggestion
          "description": "Optional, audioVideo by default."
```
Same below.